### ISSUE TITLE
Improve date picker documentation for labels

### ISSUE THREAD

**Issue by gnomeontherun** [vmware/clarity#3465]
## Describe the bug

Our date picker docs rarely use labels with their demos, and should always have a label for accessibility reasons.

## How to reproduce

See website docs https://clarity.design/documentation/datepicker

## Expected behavior

Our documentation demos and examples should be improved.

## Versions

**App**

* Angular: [e.g. 6]
* Node: [e.g. 8.10.0]
* Clarity: [e.g. 0.12.5]

**Device:**

* Type: [e.g. MacBook]
* OS: [e.g. iOS]
* Browser [e.g. chrome, safari]
* Version [e.g. 22]

## Additional notes

_Add any other notes about the problem here._

**Comment by AlexMarcuBytex** [comment]
I can add a quick fix to this issue. I would like to be assigned. Thank you!

**Comment by gnomeontherun** [comment]
Sounds good, thanks!
